dataframe-openapi/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/isOpenApi.kt

```diff
@@ -8,7 +8,7 @@ import java.net.URL
 public fun isOpenApiStr(text: String): Boolean = try {
     val parsed = OpenAPIParser().readContents(text, null, null)
     parsed.openAPI?.components?.schemas != null
-} catch (_: Exception) {
+} catch (_: Throwable) {
```

Comment:
I'm thinking, could we give more info here, just logging the reason of Error or Exception, now it silently happened

Reply:
Or, if logger is not setup, create a TODO with "add logger"

Reply:
hmm, we already have the problem of the openAPI parser logging exceptions each time we read anything with DataFrame. I don't want to needlessly log even more stuff in this case. That said, adding a logger at level INFO or DEBUG might still be beneficial indeed.

Reply:
So sad about openAPI parser logging during reading, Debug level will be a good option, so, probably a good idea to add todo or comment, to this issue or to the openAPI parser logging exception issue (if it is exists), to explain why we doing it silently, because for me it looks like anti-pattern